src/settings/toml/builder.rs

```diff
@@ -15,21 +14,30 @@ pub struct Builder {
     pub command: Option<String>,
     #[serde(default = "project_root")]
     pub cwd: PathBuf,
-    #[serde(default = "upload_dir")]
-    pub upload_dir: PathBuf,
-    pub upload_format: ScriptFormat,
-    pub upload_include: Option<Vec<String>>,
-    pub upload_exclude: Option<Vec<String>>,
     #[serde(default = "watch_dir")]
     pub watch_dir: PathBuf,
+    pub upload: UploadFormat,
```

Comment:
If I understand right now, this will look like:

```toml
[builder]
# ...
[builder.upload]
# ...
```
Would it make sense to bubble it to the top-level? The files to upload aren't really part of the "build"
```toml
[builder]
# ...
[upload]
# ...
```

Reply:
I think we should do this someday (it makes much more sense), but right now it doesn't make sense, since this would imply `[upload]` being able to sit on it's own, which implies moving `watch_dir` to top level, and there are a lot of things that need to be thought about to get right.